src/helpers/filterPastVisits.test.js

```diff
@@ -3,25 +3,30 @@ import filterPastVisits from "./filterPastVisits";
 
 describe("filterPastVisits", () => {
   beforeEach(() => {
-    MockDate.set(new Date("2020-05-18"));
+    MockDate.set(new Date("2020-05-18 13:00"));
   });
 
   afterEach(() => {
     MockDate.reset();
   });
 
-  it("returns visits that were scheduled in the past", () => {
+  it("returns visits that were scheduled between 1 hour ago and 12 hours ago", () => {
     const visits = [
-      { callTime: "2020-05-17 13:00:00" },
-      { callTime: "2020-05-17 23:59:59" },
-      { callTime: "2020-05-18 00:00:00" },
-      { callTime: "2020-05-18 00:00:01" },
+      { callTime: "2020-05-17 15:00:00" },
+      { callTime: "2020-05-18 00:59:59" },
+      { callTime: "2020-05-18 01:00:00" },
+      { callTime: "2020-05-18 09:00:00" },
+      { callTime: "2020-05-18 11:59:59" },
+      { callTime: "2020-05-18 12:00:00" },
```

Comment:
Is it worth including a visit at 12:00:01 to show what visits around this point are included ? (we have 1 second before but not 1 second after)

Reply:
Makes sense, updated the PR!